### DISCUSSION TITLE
TIDAK BISA MENGGUNAKAN KAMERA BELAKANG ( Facing Back 2,0 ) Pada Perangkat POCO X6 PRO

### DISCUSSION THREAD

**Discussion by Dikosarip62** [community/community#144379] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Saya tidak dapat menggunakan mengambil gambar dengan kamera belakang (facing back 2,0) pada perangkat POCO X6 Pro saya, tolong kepada pihak github segera memperbaiki bug pada perangkat saya🙏🙏

**Comment by ebndev** [comment]
Hi @Dikosarip62, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
